tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -170,8 +170,9 @@ shader, this value can be used as desired.
 +----------------------------------------+--------------------------------------------------------+
 | in vec3 **CAMERA_DIRECTION_WORLD**     | Camera world space direction.                          |
 +----------------------------------------+--------------------------------------------------------+
-| in bool **OUTPUT_IS_SRGB**             | ``true`` when calculations happen in sRGB color space  |
-|                                        | (``true`` in GLES2, ``false`` in GLES3).               |
+| in bool **OUTPUT_IS_SRGB**             | ``true`` when calculations happen in sRGB color space, |
+|                                        | ``false`` when calculations happen in linear color     |
+|                                        | space.                                                 |
```

Comment:
While we are at it, I would change these to "true when output is in sRGB color space (true in gl_compatibility renderer, false in RD-based renderers)."